source/agora/utils/gc/GC.d

```diff
@@ -2483,12 +2484,13 @@ struct Gcx
                                     assert(core.bitop.bt(toFree.ptr, i));
 
                                     debug(COLLECT_PRINTF) printf("\tcollecting %p\n", p);
-                                    leakDetector.log_free(sentinel_add(p));
```

Comment:
That means that `leakDetector` doesn't quite work right ?

Reply:
Leak detector is not checking for double allocations, I’ve found the problem by updating `log_malloc`. I can also send that with the PR.